src/functional_test_runner/cli.js

```diff
@@ -21,10 +20,8 @@ cmd
   .parse(process.argv);
 
 if (!cmd.config) {
-  console.log('');
-  console.log('  --config is a required parameter');
-  console.log('');
-  process.exit(1);
+  const [project = 'functional'] = cmd.args;
+  cmd.config = resolveConfigPath(`test/${project}/config.js`);
 }
```

Comment:
The idea here was to introduce "projects", but I later decided that it's not time and that it's too big a breaking change to not put more thought into.